WCF/Shared/RequestTrackingTelemetryModule.cs

```diff
@@ -44,7 +44,7 @@ void IWcfTelemetryModule.OnBeginRequest(IOperationContext operation)
             var httpHeaders = operation.GetHttpRequestHeaders();
             if ( httpHeaders != null )
             {
-                telemetry.HttpMethod = httpHeaders.Method;
+                telemetry.Properties["httpMethod"] = httpHeaders.Method;
```

Comment:
The problem is that for WCF that doesn't make sense:

* For non-HTTP bindings, there's no verb
* For HTTP-based bindings, it's always going to be POST, so it only clutters the output.

The only case where it turns out to be relevant is when using webHttpBinding, which is used for REST services; I do want to improve that, but there's no easy way to detect that is the case here, and in most cases the verb would only give you something that the real operation name as we already report gives you.

Right now for WCF, the operation name == <ContractName>.<OperationName> which is far, far more useful than the URL (which won't allow you to tell different operations apart).

Thoughts?